src/domain/transfer/commands/index.js

```diff
@@ -20,52 +20,92 @@
  --------------
  ******/
 
-'use strict'
+// 'use strict'
```

Comment:
why is this whole file commented out? is there a need for it? if not, it should be removed

Reply:
removed the file